app/mainRoutes.jsx

```diff
@@ -12,9 +12,11 @@ module.exports = (
 	<Route name="app" path="/" handler={require("./Application")}>
 		<Route name="some-page" path="/some-page" handler={require("react-proxy!./SomePage")} />
 		<Route name="readme" path="/readme" handler={require("react-proxy!./Readme")} />
-		<Route name="todolist" path="/:list" handler={require("./TodoList")} />
-		<Route name="todoitem" path="/todo/:item" handler={require("./TodoItem")} />
+		<Route name="todo" path="/todo" handler={require("react-proxy!./TodoPage")} >
```

Comment:
Try to no use `react-proxy` here. This shouldn't be loaded on demand. (On demand loaded stuff cannot be prerendered.)